sqlalchemy-stubs/orm/decl_m.pyi

```diff
@@ -0,0 +1,236 @@
+from .session import Session as Session
+from .session import sessionmaker as sessionmaker
+from .session import SessionTransaction as SessionTransaction
+from .state import AttributeState as AttributeState
+from .state import InstanceState as InstanceState
+from .strategy_options import Load as Load
+from .unitofwork import UOWTransaction as UOWTransaction
+from .util import aliased as aliased
+from .util import Bundle as Bundle
+from .util import CascadeOptions as CascadeOptions
+from .util import join as join
+from .util import LoaderCriteriaOption as LoaderCriteriaOption
+from .util import object_mapper as object_mapper
+from .util import outerjoin as outerjoin
+from .util import polymorphic_union as polymorphic_union
+from .util import was_deleted as was_deleted
+from .util import with_parent as with_parent
+from .util import with_polymorphic as with_polymorphic
+from ..sql import schema
+from ..sql import sqltypes
+from ..sql.schema import _ServerDefaultType
+from ..sql.schema import FetchedValue
+from ..sql.schema import SchemaEventTarget
+
+_T = TypeVar("_T")
+
+_BackrefResult = Tuple[str, Mapping[str, Any]]
+
+class MappedColumn(Mapped[_T]):
+    def __init__(self, expr: _T) -> None: ...
+    def operate(self, op: Any, *other: Any, **kwargs: Any): ...
+    def reverse_operate(self, op: Any, other: Any, **kwargs: Any): ...
+
+@overload
+def column(
+    __name: str,
+    *args: SchemaEventTarget,
+    autoincrement: Union[bool, Literal["auto", "ignore_fk"]] = ...,
+    default: Optional[Any] = ...,
+    doc: Optional[str] = ...,
+    key: Optional[str] = ...,
+    index: Optional[bool] = ...,
+    info: MutableMapping[Any, Any] = ...,
+    nullable: bool = ...,
+    onupdate: Optional[Any] = ...,
+    primary_key: bool = ...,
+    server_default: Optional[_ServerDefaultType[Any]] = ...,
+    server_onupdate: Optional[FetchedValue] = ...,
+    quote: Optional[bool] = ...,
+    unique: Optional[bool] = ...,
+    system: bool = ...,
+    comment: Optional[str] = ...,
+    **kwargs: Any,
+) -> Mapped[Any]: ...
+@overload
+def column(
+    *args: SchemaEventTarget,
+    autoincrement: Union[bool, Literal["auto", "ignore_fk"]] = ...,
+    default: Optional[Any] = ...,
+    doc: Optional[str] = ...,
+    key: Optional[str] = ...,
+    index: Optional[bool] = ...,
+    info: MutableMapping[Any, Any] = ...,
+    nullable: bool = ...,
+    onupdate: Optional[Any] = ...,
+    primary_key: bool = ...,
+    server_default: Optional[_ServerDefaultType[Any]] = ...,
+    server_onupdate: Optional[FetchedValue] = ...,
+    quote: Optional[bool] = ...,
+    unique: Optional[bool] = ...,
+    system: bool = ...,
+    comment: Optional[str] = ...,
+    **kwargs: Any,
+) -> Mapped[Any]: ...
+@overload
+def column(
+    __name: str,
+    _type: Union[sqltypes.TypeEngine[_T], Type[sqltypes.TypeEngine[_T]]],
+    *args: SchemaEventTarget,
+    autoincrement: Union[bool, Literal["auto", "ignore_fk"]] = ...,
+    default: Optional[Any] = ...,
```

Comment:
Can this be Optional[T] ?

Reply:
Maybe this isn't clear from the github interface, but I meant "default: Optional[T] = ..."